server/block/wood_sign.go

```diff
@@ -0,0 +1,131 @@
+package block
+
+import (
+	"fmt"
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/block/model"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/go-gl/mathgl/mgl64"
+	"math"
+	"strings"
+)
+
+// WoodSign is block that you can write text on.
+type WoodSign struct {
+	transparent
+	bass//might be wrong
+
+
+	// Wood is the type of wood of the door. This field must have one of the values found in the material
+	// package.
+	Wood WoodType
+	// Pos is the position of the sign and the block entity
+	Pos cube.Pos
```

Comment:
Fields that aren't saved in block NBT or states won't persist properly, so this probably won't as expected. See my hint above. You'll probably want to remove this field.